ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -1165,8 +1165,9 @@ export class DSMService {
 
   private checkCookieBeforeCall(): boolean {
     if (this.sessionService.getDSMToken() == null) {
+      this.localStorageService.clear();
```

Comment:
Can't import the auth service because of a cyclical dependency, so logic handles the updates needed.